pnpm-lock.yaml

```diff
@@ -1256,8 +1256,8 @@ importers:
         specifier: ^2.8.8
         version: 2.8.8
       prisma:
-        specifier: https://registry.npmjs.com/prisma/-/prisma-5.8.1.tgz?id=%40examplesnext-websockets-starter
-        version: '@registry.npmjs.com/prisma/-/prisma-5.8.1.tgz?id=%2525252525252525252525252525252525252525252525252525252525252525252525252525252540examplesnext-websockets-starter'
```

Comment:
this stopped happening on every install when dropping the `@` sign in the examples

likely a bug in pnpm